content/post/2018-06-05_badcomkillsgoodanalytics.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Randy Clinton
+title: "Bad communication kills good analytics"
+cover: MISSING
+publishDate: UNKNOWN
+tags:
+ - teams
+ - communication
+ - presentation
+---
+
+Poor communication within an Analytics team and between that team and the company, leave highly skilled analytics employees solving the wrong questions, lacking support for big ideas and leaving the company unsatisfied with their work. Here, I'll discuss ways a team can improve by focusing on constructive conversions.
+<!--more-->
+
+Analytics team exist because companies wants better conversations. Technical tools allow us ways to find information, but at the end we need to talk about it. Without an analytics team, a company will discuss business decisions like marketing spending, product offerings, and ways to encourage customers repurchase. A productive analytics team grounds these conversations in truth and testing.
+
+Four constructive conversations distinguish high functioning analytics teams:
+
+#### Constructive Conversations
+* Talk as a team
+* Talk about your work
+* Talk about the business
+* Talk about where the data is buried
+
+## Talk as a team
+Each week your company has a number of people digging into data, reading documentation for various programming language modules, and attempting to solve business questions.
+
+Teams focusing on talking together spend less time re-learning the same thing. Their members spend less time being frustrated in solitude, and they enjoy their work because they’re learning more.
+
+#### Improving “talk as a team”
+
+###### Crowdsource “I’m stuck”
+Create a safe space to vent. It can be a team meeting or a private channel for just the analytics team, but this is crucial to embracing and learning from failure. You’re asking team members to be vulnerable, so set the example in these places by sharing your struggles. The responses should be supportive and in-line with a team committed to growing together.
+
+###### Code Review
+Include a process for documenting and reviewing work in your team’s definition of “done.” The common definition of “done” reminds the team what they’ve agreed to produce. During the code review, the reviewer gets a walk through exactly what, why and how this was done. Plus, the reviewing affirms best practices for the team. This will look different than a traditional code review for engineers because you want to review the business logic and steps of analysis as much as the code.
```

Comment:
Maybe rephrase "Plus, the reviewing affirms best practices for the team." to

> Additionally, good code review reinforces best practices within the team.

Reply:
Also "This will look different than a traditional code review for engineers because you want to review the business logic and steps of analysis as much as the code." could become 
>Code review for analytics is often substantively different from code review for software engineering because reviewers need to check the business logic and the analytical methods as well as the code.

Reply:
ooh i like that better thanks.